cspell.json

```diff
@@ -90,6 +91,8 @@
     "numbervisibilityprocess",
     "OAUTHCONFIG",
     "oidc",
+    "overridability",
```

Comment:
there's no such word in English, maybe should be replaced with something different?

Reply:
it is sourced from Wiktionary, is that even a thing? anyway, this needs the spellcheck exception list editing, so not recognized somewhere as a word?

Reply:
If it's ok for you, I'll replace `Overridability` with `Override` in class/file names, cspell is ok with it.
Will remove this entry ofc

Reply:
sure anything that will be dead simple will do... we have a lot of people that are non-English speakers, so not to confuse everyone, thanks